site/source/docs/api_reference/html5.h.rst

```diff
@@ -927,6 +927,10 @@ Defines
     Emscripten `orientationchange <https://w3c.github.io/screen-orientation/>`_ event.
 
 
+.. c:macro:: EMSCRIPTEN_ORIENTATION_UNSUPPORTED
+
+  The orientation API is not supported and the type is invalid.
```

Comment:
What does "and the type is invalid" mean here?   What type are we talking about?

Reply:
The orientation type, which I realise in the Emscripten API the field is called `orientationIndex` (JS calls it `type`). I wanted to get across that the value isn't a valid orientation. I removed that part.

Reply:
I see in that case should this be "or the orientation type is is not supported by emscripten"?

Should this be *or* instead of *and*?

Reply:
I rewrote this and added more to the `orientationIndex` field.

Reply:
(And changing the docs file appears to have killed the CI, I can only assume they're not related looking at the errors)